ui/app/src/Utils/Types.elm

```diff
@@ -1,20 +1,19 @@
 module Utils.Types exposing (..)
 
-import Http
 import Time
 
 
-type ApiResponse e a
+type ApiData a
     = Initial
     | Loading
-    | Failure e
+    | Failure String
```

Comment:
I don't see where we require this information. I need to create the error myself, when e.g. validation fails. It is easier to create it from `String`, rather than manually create `Http.Error`, I don't think this is even possible.